hw/ip/usbdev/data/usbdev.hjson

```diff
@@ -150,6 +150,13 @@
       default: "12",
       desc:    "Number of endpoints",
       local:   "true"
+    },
+    { name:    "RcvrWakeCycles",
```

Comment:
do you think it makes more sense to express this in terms of uS?
I remember there is a uS tick already in the `usbdev`.

It doesn't make a huge difference since I think for this design we ALWAYS need a 48MHz clock, so expressing it in terms of the clock is fine.  But expressing it in terms of time without knowing the cycle feels a bit more abstracted.

Reply:
Yeah, microseconds seems fine to me. I'll make that change.